capstone/src/main/webapp/script.js

```diff
@@ -34,21 +34,36 @@ function configureSearchResponseToHTML(studySets) {
   return configuredStudySetInfoCards;
 }
 
+function encodeHTML(str) {
+  if (str === null || str === "") return false;
+  else str = str.toString();
+  return str.replace(/<[^>]*>/g, "");
```

Comment:
As discussed in chat, this isn't a safe way to escape HTML tag. For example:

```
<script
  src="some-malicious-script.com"
/>
```

Line break avoids the regex. Remove this code for now, adding code with known security vulnerabilities just adds confusion for future maintainers.